src/gamecontroller.cs

```diff
@@ -1922,4 +1996,50 @@ enum GCInputDirectional {
 		[Field ("GCInputDirectionalCardinalDpad")]
 		CardinalDpad,
 	}
+
+	delegate GCVirtualControllerElementConfiguration GCVirtualControllerElementUpdateBlock (GCVirtualControllerElementConfiguration configuration);
+
+	[NoTV, NoMac, NoWatch, iOS (15,0), MacCatalyst (15,0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
```

Comment:
@rolfbjarne I know you already reviewed this PR but I just wanted to make sure that this line is correct. Also for others reviewing later:
I added this since there didn't appear to be an initializer for this interface and we have the `virtualControllerWithConfiguration:` that creates an instance

Reply:
Yes, this looks entirely correct!

Also, if in doubt, it's better to put this attribute than to not to, because we can always add the default ctor later (but we can't remove it once we've made it a part of the stable API).